sway_libs/src/nft/extensions/meta_data/meta_data.sw

```diff
@@ -10,11 +10,12 @@ pub trait MetaData<T> {
     /// Returns the metadata for this token
     #[storage(read)]
     fn meta_data(self) -> Option<T>;
+
     /// Creates new metadata for this token.
     ///
     /// # Arguments
     ///
-    /// * `metadata` - The metadata value to be included in the new metadata.
+    /// * `metadata` - The new metadata to overwrite the existing metadata.
```

Comment:
This is not useful. If I do not know what `metadata` is meant to be then telling me that it's "metadata that overwrites the old one" doesn't convey any semantic information about what `metadata` actually is.

Reply:
Good point